server/build.gradle

```diff
@@ -49,7 +49,7 @@ configurations
         }
 configurations.binaries.setDescription("Utility and proteomics binaries")
 
-if (!project.hasProperty('excludeProteomicsBinaries'))
+if (project.hasProperty('includeProteomicsBinaries'))
```

Comment:
You could do this the opposite way by adding the exclude property to the `gradle.properties` file and leaving this as is, which would mean no need to update docs or potentially communicate the switch to anyone.  I guess this way forces someone to make the decision again to choose them and might make it easier to remove altogether, but please do update the [documentation](https://www.labkey.org/Documentation/wiki-page.view?name=gradleTips) and at least let the support team know of the change.

Reply:
I was going to do that but wanted to retain the ability to control the property without modifying any files. It was either a new property or make `excludeProteomicsBinaries` take a value, which would also require documentation. This seemed the least invasive change, presuming that a majority of developers don't need the binaries.